content/post/090-go-ipfs-0-5-0.md

```diff
@@ -0,0 +1,102 @@
+---
+date: 2020-04-28
+url: 2020-04-28-go-ipfs-0-5-0
+title: IPFS 0.5.0 is here! Our largest upgrade to IPFS yet.
+author: Molly Mackinlay
+header_image: 090-go-ipfs-0-5-0.png
+tags: go-ipfs, release
+---
+![Announcing IPFS 0.5](static/header_images/090-go-ipfs-0-5-0.png)
+
+*go-ipfs 0.5.0 is a major step forward on our journey toward a more efficient, secure, and resilient web to preserve and grow humanity’s knowledge.* 
+
+We’re happy to share that IPFS 0.5.0, the largest upgrade to the IPFS protocol and Public Network yet, is now available. The result of extensive community feedback and rigorous focus, IPFS 0.5.0 offers major performance and reliability updates so that you can distribute content peer-to-peer more efficiently and affordably than ever before.
```

Comment:
```suggestion
We’re happy to share that IPFS 0.5.0, the largest upgrade to the IPFS protocol and Public Network yet, is now available. The result of extensive community feedback and rigorous focus, IPFS 0.5.0 offers major performance and reliability updates so that you can distribute content peer-to-peer more efficiently and affordably.
```